routes.js

```diff
@@ -68,4 +70,35 @@ module.exports = function (app, router) {
 
     //route for getting pokemon details by specified query parameters
     router.get('/pokemon/search', pokemon.search);
+
+    // route for getting sentiments about a given pokemon
+    router.get('/pokemon/:pokemonNumber/sentiments', function (req, resp) {
+        pokemon.sentiments(req, resp, getMongoDbUrl());
+    });
+
+    // route for getting sentiments about a given pokemon at a given location
+    router.get('/pokemon/:pokemonNumber/sentiments/:lat/:lng', function (req, resp) {
+        pokemon.sentiments(req, resp, getMongoDbUrl());
+    })
 };
+
+/**
+ * Reads the config to compute the mongodb database url (without collection as part of the url)
+ */
+function getMongoDbUrl() {
+    var dbConnection = "mongodb://";
+    var databaseParams = appConfig['IS_LOCAL_DB'] ? config['database'] : config['shared_database'];
+
+    // check whether there is username and password for database connection
+    if (databaseParams.username && databaseParams.password) {
+        dbConnection += databaseParams.username + ":" + databaseParams.password + "@";
+    }
+
+    //Connection parameters for a local database instance
+    if (appConfig['IS_LOCAL_DB'])
+        dbConnection += databaseParams.uri + ":" + databaseParams.port;
+
+    //Connection parameters for a shared database instance
+    else
+        dbConnection += databaseParams.uri;
+}
```

Comment:
This function does not seem to return anything and is being called above. Maybe `dbConnection` was intended to be returned.

Also, this functionality already exists in `database.js`. So it could be extracted into a function and just invoked from above.